portfolio/src/main/webapp/style.css

```diff
@@ -13,18 +13,46 @@
     height: 250px;
 }
 
-.image-thumbnails {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;  
+.image-container {
+    column-count:3;
+    margin: 0 auto;
+    padding: 30px;
 }
   
-.image-thumbnails img {
-    margin: 10px;
-    max-height: 200px;
-    max-width: 200px;
+.image-container img {
+    max-width:100%;
+    max-height:100%;
+    width: auto;
+    height: auto;
+    padding:10px;
+    box-sizing: border-box;
+    
 }
   
-.image-thumbnails img:hover {
+.image-container img:hover {
     border: thick solid black;
+}
+
+#info-container{
+    display:flex;
```

Comment:
prefer a space after the CSS rule name ("display: flex").

Reply:
Done